src/frontend/get-github-configuration.ts

```diff
@@ -111,45 +112,73 @@ const removeFailedConnectionsFromDb = async (req: Request, installations: any, j
 
 export default async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 	const { jiraHost, githubToken } = req.session;
+	const log = req.log.child({ jiraHost });
+
+	const traceLogsEnabled = await booleanFlag(BooleanFlags.TRACE_LOGGING, false);
```

Comment:
Shouldn't jiraHost be included to `booleanFlag` call?

Reply:
no, I explicitly want to make this a global flag, because I'm not sure if the `jiraHost` is available in all places.